govc/vcsa/access/dcui/set.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright (c) 2022 VMware, Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0.
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dcui
+
+import (
+	"context"
+	"flag"
+
+	"github.com/vmware/govmomi/govc/cli"
+	"github.com/vmware/govmomi/govc/flags"
+	"github.com/vmware/govmomi/vapi/appliance/access/dcui"
+)
+
+type set struct {
+	*flags.ClientFlag
+	dcuiEnabled bool
+}
+
+func init() {
+	cli.Register("vcsa.access.dcui.set", &set{})
+}
+
+func (cmd *set) Register(ctx context.Context, f *flag.FlagSet) {
+	cmd.ClientFlag, ctx = flags.NewClientFlag(ctx)
+	cmd.ClientFlag.Register(ctx, f)
+
+	f.BoolVar(&cmd.dcuiEnabled,
+		"dcui-enabled",
+		false,
+		"Enable Direct Console User Interface (DCUI TTY2).")
+}
+
+func (cmd *set) Process(ctx context.Context) error {
+	if err := cmd.ClientFlag.Process(ctx); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (cmd *set) Description() string {
+	return `Set enabled state of Direct Console User Interface (DCUI TTY2).
```

Comment:
nit: "Set state..."

Reply:
As per offline discussion, we agreed to keep the wordings the same as these wordings were copied from the official API doc.